src/chrome/content/rules/Becu.org.xml

```diff
@@ -0,0 +1,17 @@
+<ruleset name="Becu.org">
+    <target host="becu.org" />
+    <target host="www.becu.org" />
+    <target host="onlinebanking.becu.org" />
+    <target host="onlineappointments.becu.org" />
+    <target host="accessassistant.becu.org" />
+
+    <exclusion pattern="^http://news\.becu\.org/" />
+    <exclusion pattern="^http://locatorsearch\.becu\.org/" />
```

Comment:
You don't need exclusions or test for non existent targets.
Please make a comment on the top an document their failures.